config.go

```diff
@@ -25,6 +29,16 @@ func LoadConfigFromPath(path string) (Config, error) {
 	if dsn, found := os.LookupEnv("TIGERBLOOD_DSN"); found {
 		config.DatabaseDsn = dsn
 	}
+	if dbMaxOpenConns, found := os.LookupEnv("TIGERBLOOD_DB_MAX_OPEN_CONNS"); found {
+		dbMaxOpenConns, err := strconv.Atoi(dbMaxOpenConns)
+		if err != nil {
+			return config, err
```

Comment:
Should append to the error to make it more clear. For example:

`return config, fmt.Errorf("Error parsing TIGERBLOOD_DB_MAX_OPEN_CONNS: %v", err)`